Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "soda-test-service"
-version = "0.1.3"
+version = "0.1.4"

```

The version should be 0.2.0 since we add a new command line argument, we change the API.
```suggestion
version = "0.2.0"
```